server/lib/enrich/article.js

```diff
@@ -31,6 +31,8 @@ module.exports = exports = function article(content, format) {
 		content.bodyHTML = content.body;
 	}
 
+	content.hasGraphics = content.contentStats && content.contentStats.graphics > 0;
+
```

Comment:
if `contentStats` is undefined you'll get undefined:
```
> content = {}
{}
> content.contentStats && content.contentStats.graphics > 0
undefined
```

Reply:
Good spot! 👍